src/main/java/com/endava/soa_5ed/task2/arrays/CodeReadingTask.java

```diff
@@ -0,0 +1,75 @@
+package com.endava.soa_5ed.task2.arrays;
+
+public class CodeReadingTask {
+    /* Arrays - Exercise 1 */
+    public static int multiplyArrayNumbers(int[] a) {
+        //this method multiplies all the numbers in an array and returns the product
+        int product = 1;
+        int i = 0;
+        while (i < a.length) {
```

Comment:
You way use here ++i < a.length and remove line 11

Reply:
wouldn't it just skip the first item, then?

Reply:
wouldn't it just skip the first element in the array?